gneiss/tests/test_util.py

```diff
@@ -0,0 +1,225 @@
+import unittest
```

Comment:
I realized that none of this files have the copyright notice on top. Should it be added?